.github/workflows/go.yaml

```diff
@@ -71,6 +72,21 @@ jobs:
         # root ownership so the tests must be run as root:
         run: sudo make go.unit
 
+      - name: dqlite-for-golangci-lint
+        working-directory: src/k8s
+        run: |
+          sudo add-apt-repository ppa:dqlite/dev
+          sudo apt update
+          sudo apt install dqlite-tools libdqlite-dev
+
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v6
+        with:
+          version: v1.61
+          working-directory: src/k8s
+          args: '-c ./../../golangci.yml'
+          only-new-issues: true
```

Comment:
TBH I expected us to get blown away by the sheer amount of issues that adding this linter would report, but I can see that the linter was executed without any finding any issues. Is this by any means because of the `only-new-issues` option?

Reply:
yes, we don't want that. If we enable this linter then we want the whole code to comply. 

Please disable that option.